### DISCUSSION TITLE
rename them with a set of individual names

### DISCUSSION THREAD

**Discussion by apexSolarKiss** [75lb/renamer#60] [Q&A]
I am looking to take a set of identical png files and rename them with a set of individual names

I think I might be able to do this with renamer, but not fully sure how…

say I would make a text file _names.txt_ >>
_apple.png
banana.png
…
orange.png_

Looking to run this on the following 200 files >>
_file1.png
file2.png
…
file200.png_

Thank you in advance for any insights

**Comment by 75lb** [comment]
so, to be clear, you want `file1` to become `apple`, `file2` to become `banana` etc?

**Reply by apexSolarKiss** [reply]
yes, correct

(all with .png extensions)

**Comment by 75lb** [comment]
I'll put together an example plugin showing how this is done - probably tomorrow (UK time)
